src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/BatchExecutionTest.java

```diff
@@ -235,7 +235,11 @@ private static void dropTable() throws SQLException {
     }
 
     @AfterAll
-    public static void terminateVariation() throws SQLException {
+    public static void terminateVariation() throws Exception {
+        try (DBConnection con = new DBConnection(connectionString)) {
```

Comment:
do we need to repeat server version check in both BeforeAll and AfterAll? the check from AfterAll will always be redundant.

Reply:
This is not redundant, in new Junit version, before all and after all are called even when one of them is skipped. If you remove these checks, the tests always fail. Check builds before [this](https://github.com/Microsoft/mssql-jdbc/pull/807/commits/7a97eba07e983c24324913b793c9b69c504fb207) commit.